src/dos/program_imgmount.cpp

```diff
@@ -269,11 +275,13 @@ void IMGMOUNT::Run(void) {
         std::vector<DOS_Drive*>::size_type ct;
 
         for (i = 0; i < paths.size(); i++) {
-            std::unique_ptr<fatDrive> newDrive(
-                new fatDrive(paths[i].c_str(),sizes[0],sizes[1],sizes[2],sizes[3],0));
+		std::unique_ptr<fatDrive> newDrive(
```

Comment:
Something doesn't look right with this indentation.  Looks like it might be a single space inside the for loop.

Reply:
Good eye @bmunger .
Looks like the `for` and its content need to be tabbed in:

![2022-01-06_17-29](https://user-images.githubusercontent.com/1557255/148476456-5b88e158-1ac2-46b7-88b0-442efdedeee6.png)